diagnostic.sql

```diff
@@ -5,24 +5,45 @@ CREATE DATABASE "sql-diagnostic";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname, given_name, gender, height, weight, born_on, eye_color, hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+-- CREATE TABLE licensees {
```

Comment:
`--` Creates a comment in SQL, causing the following command to result in an error.